Participants/QuantStack.tex

```diff
@@ -11,6 +11,13 @@ \subsubsection*{Curriculum vitae of the investigators}
 \input{CVs/Martin.Renou.tex}
 \input{CVs/Wolf.Vollprecht.tex}
 
+% For other to-be-hired person, please include here something like:
+\begin{participant}[type=res,PM=41,salary=7000]{NN} %% Salary: standard SME cost
```

Comment:
specified 41 PM (since 6 PM are allocated on myself). the salary is the standard SME cost.